app/assets/javascripts/listings/components/listingContainer.js.coffee

```diff
@@ -2,85 +2,82 @@ angular.module('dahlia.components')
 .component 'listingContainer',
   transclude: true
   templateUrl: 'listings/components/listing-container.html'
-  controller: [
-    'ListingService', 'ListingHelperService', 'SharedService',
-    (ListingService, ListingHelperService, SharedService) ->
-      ctrl = @
-      # TODO: remove Shared Service once we create a Shared Container
-      @listingEmailAlertUrl = "http://eepurl.com/dkBd2n"
-      @assetPaths = SharedService.assetPaths
-      @listing = ListingService.listing
-      @listings = ListingService.listings
-      @loading  = ListingService.loading
-      @error = ListingService.error
-      @toggleStates = ListingService.toggleStates
-      @AMICharts = ListingService.AMICharts
-      @favorites = ListingService.favorites
-      @listingDownloadURLs = ListingService.listingDownloadURLs
+  controller: ['ListingDataService', 'ListingEligibilityService', 'ListingIdentityService', 'ListingUnitService', 'SharedService',
+  (ListingDataService, ListingEligibilityService, ListingIdentityService, ListingUnitService, SharedService) ->
+    ctrl = @
+    # TODO: remove Shared Service once we create a Shared Container
+    @listingEmailAlertUrl = "http://eepurl.com/dkBd2n"
```

Comment:
I wonder if this should be moved to the constants service

Reply:
yeah i think so. this is another topic we should discuss: the intention around location of data, and also what the distinction is between business logic and display logic and the locations of those things, which related a bit to the question you had the other day. adding these to the PR discussion topics